server/pom.xml

```diff
@@ -265,6 +265,11 @@
             <artifactId>JUnitParams</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
```

Comment:
this library is not Apache v2.0 , is that ok given that we are only going to use it in tests but it wouldn't be packaged in final distribution.

Reply:
For reference, the license is CPLv1.0: https://github.com/stefanbirkner/system-rules/blob/master/LICENSE

I found this thread that discusses 3rd party licenses in "test" code: https://issues.apache.org/jira/browse/LEGAL-207

Reply:
The ASF guidance for "weak copyleft" licenses (like CPLv1.0) requires the inclusion to be appropriately labeled:
- https://apache.org/legal/resolved.html#weak-copyleft-licenses
- https://apache.org/legal/resolved.html#appropriately-labelled-condition

Note that EPLv1.0 is a "weak copyleft" license and is the license for JUnit4: https://junit.org/junit4/license.html. I can't find anything in the druid repo that satisfies the "Appropriate labelled condition" for JUnit4.

Reply:
This library will not be included in source or binary distribution because we don't bundle tests or benchmarks so shouldn't matter.

Reply:
We don't have anything for JUnit since we don't include any source from JUnit and it's not bundled in our binary distribution.

For the weak copyleft licenses like EPLv1.0, the "labeling" is handled in a section at the end of our README (See README.BINARY), and we'll need to add the CPL license for this under `licenses/bin`

Reply:
Since this is not bundled with the binary distribution and we don't include any of the source (only as a test dependency), I don;t think we have to do anything

Reply:
@ccaominh @clintropolis @jon-wei thanks for finding all the info.